stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,86 @@
+{{- $svcname := "weave-scope" -}}
+{{- $svcport := 80 -}}
+
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    {{- include "weave-scope.helm_std_labels" . | indent 4 }}
+    component: agent
+  name: {{ template "weave-scope.fullname" . }}-agent
+  annotations:
+    {{- include "weave-scope.annotations" . | indent 4 }}
+spec:
+  selector: 
+    matchLabels:
+      app: {{ template "weave-scope.name" . }}
+      release: {{ .Release.Name }}
+      component: agent
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        {{- include "weave-scope.helm_std_labels" . | indent 8 }}
+        component: agent
+    spec:
+      tolerations:
+        - effect: NoSchedule
+          operator: Exists
+      containers:
+        - name: agent
+          image: {{ required "You must specify the desired container image" .Values.global.image }}:{{ required "You must specify the desired container image tag" .Values.global.imageTag }}
+          {{- if .Values.imagePullPolicy }}
```

Comment:
Remove the if statement and set `IfNotPresent` in `values.yaml`.

Reply:
Doc change needed?  [Chart Best Practices](https://docs.helm.sh/chart_best_practices/#imagepullpolicy) says it should default to an empty value.

Reply:
Interesting. I wasn't aware of that. Chart examples and the chart created by `helm create` use `IfNotPresent`.

Reply:
I filed kubernetes/helm#3099 on this.